bin/tessel-2.js

```diff
@@ -19,9 +19,12 @@ const CLI_ENTRYPOINT = 'cli.entrypoint';
 
 // Check for updates
 const pkg = require('../package.json');
-updateNotifier({
-  pkg
-}).notify();
+
+try {
+  updateNotifier({ pkg }).notify();
+} catch (err) {
+  CrashReporter.submit(err.stack, { silent: true });
```

Comment:
So, now it's quiet for the user, but http://crash-reporter.tessel.io/ is going to continue to amass these reports. I think the real fix is to simply suppress this error :| (or maybe the authors of `configstore` could just make the damn thing work without barfing all over the place